source/GameInterface/AutoSync/Properties/PropertySwitchCreator.cs

```diff
@@ -155,7 +155,7 @@ private void CreateByValue(ILGenerator il, PropertyInfo property, LocalBuilder i
         il.Emit(OpCodes.Call, AccessTools.Method(typeof(PropertySwitchCreator), nameof(Deserialize)).MakeGenericMethod(property.PropertyType));
 
         il.Emit(OpCodes.Call, AccessTools.Method(typeof(AllowedThread), nameof(AllowedThread.AllowThisThread)));
-        il.Emit(OpCodes.Callvirt, property.GetSetMethod());
+        il.Emit(OpCodes.Callvirt, property.GetSetMethod(true)); // Joke pls fix
```

Comment:
Should be fixed on main once #1000 is completed

Reply:
Fixed indeed, since #1000 was already merged I'll just rebase onto dev again